src/toolkits/object_detection/od_data_iterator.cpp

```diff
@@ -50,7 +53,10 @@ gl_sframe get_data(const data_iterator::parameters& params) {
 std::vector<image_annotation> parse_annotations(
     const flex_list& flex_annotations,
     size_t image_width, size_t image_height,
-    const std::unordered_map<std::string, int>& class_to_index_map) {
+    const std::unordered_map<std::string, int>& class_to_index_map,
+    image_origin_enum image_origin_, 
```

Comment:
Nit: for function arguments (and other local variables), let's use names that do not end in underscore, so `annotation_scale` instead of `annotation_scale_`

The underscore should be a signal that the variable is a data member (instance variable)

Reply:
They are data members of the `simple_data_iterator` class and I'm passing them in `parse_annotations`. Maybe I don't need to pass them to use them..

Reply:
Yes, the arguments you pass to `parse_annotations` when you call it may have underscores at the end, because the arguments are data members of `simple_data_iterator`, but the named arguments for the function itself are not data members. They are local variables that happen to be initialized (at the call site) by those data members.

Suppose I have a function `add_one` that adds one to its argument, and I use it to increment a counter (that is a data member):
```
  counter_ = add_one(counter_);
```

Does that mean that the name of the argument in the definition of `add_one` should be `counter_`? What makes more sense:
```
  int add_one(int x) { return x + 1; }
```
or
```
int add_one(int counter_) { return counter_ + 1; }
```
?

Reply:
Ohhh - definitely the first one. Thank you for explaining this.